tests/migrations/test_state.py

```diff
@@ -1016,6 +1018,23 @@ class Meta:
         choices_field = Author._meta.get_field('choice')
         self.assertEqual(list(choices_field.choices), choices)
 
+    def test_warn_callable_defaults(self):
+        new_apps = Apps(['migrations'])
+
+        class Author(models.Model):
+            random_seed = models.FloatField(unique=True, default=random.Random)
+
+            class Meta:
+                app_label = 'migrations'
+                apps = new_apps
+
+        msg = 'Callable defaults will not generate unique values upon migrating. '
+        msg += "Found callable default <class 'random.Random'> on unique field 'Author.random_seed'. "
+        msg += 'See https://docs.djangoproject.com/en/dev/howto/writing-migrations/#migrations-that-add-unique-fields'
```

Comment:
Something like this should work? 

```suggestion
        msg = (
            'Callable defaults will not generate unique values upon migrating. '
            "Found callable default <class 'random.Random'> on unique field 'Author.random_seed'. "
            'See https://docs.djangoproject.com/en/dev/howto/writing-migrations/#migrations-that-add-unique-fields'
        )
```